### PR TITLE
Info update - Discord Linking

### PR DESCRIPTION
**Pull request by Raduyno**

### CODE REVIEW

File: src/main/java/me/unraveledmc/unraveledmcmod/discord/MessageListener.java [hunk 1]
```diff
@@ -22,6 +22,7 @@ public void onPrivateMessageReceived(PrivateMessageReceivedEvent event)
                     admin.setDiscordID(event.getMessage().getAuthor().getId());
                     Discord.LINK_CODES.remove(code);
                     Discord.sendMessage(event.getChannel(), "Successfully linked this discord account to the minecraft account `" + admin.getName() + "`");
+                    Discord.sendMessage(event.getChannel(), "Now when you are a imposter you may use `/verify` to verify.");
```
Comment (ItzLevvie) [inline src/main/java/me/unraveledmc/unraveledmcmod/discord/MessageListener.java]: Should be ```impostor``` instead of ```imposter```

Comment (Raduyno) [inline src/main/java/me/unraveledmc/unraveledmcmod/discord/MessageListener.java]: Hm. Ok.

File: src/main/java/me/unraveledmc/unraveledmcmod/discord/MessageListener.java [hunk 2]
```diff
@@ -22,6 +22,7 @@ public void onPrivateMessageReceived(PrivateMessageReceivedEvent event)
                     admin.setDiscordID(event.getMessage().getAuthor().getId());
                     Discord.LINK_CODES.remove(code);
                     Discord.sendMessage(event.getChannel(), "Successfully linked this discord account to the minecraft account `" + admin.getName() + "`");
+                    Discord.sendMessage(event.getChannel(), "Now when you are a impostor you may use `/verify` to verify.");
```
Comment (OxLemonxO) [inline src/main/java/me/unraveledmc/unraveledmcmod/discord/MessageListener.java]: `an` not `a`

Comment (LightWarp) [inline src/main/java/me/unraveledmc/unraveledmcmod/discord/MessageListener.java]: rip grammar

**Comment by CreeperSeth** [comment]
Merged. Thank you for your contribution.

**Pull request merged**